package.json

```diff
@@ -55,7 +55,7 @@
   "license": "MIT",
   "repository": {
     "type": "git",
-    "url": "https://github.com/fastify/busboy.git"
+    "url": "git+https://github.com/fastify/busboy.git"
```

Comment:
where does this syntax come from? I don't see it in https://docs.npmjs.com/cli/v10/configuring-npm/package-json#repository

Reply:
@kibertoad 🤷 comes from running `npm pkg fix` using npm 10.4.0.

Reply:
npm gonna npm. probably they changed something and didn't update the docs accordingly